engines/bops_api/lib/bops_api/schemas.rb

```diff
@@ -22,6 +22,14 @@ def find!(name, version: nil, schema: nil)
         raise BopsApi::Errors::SchemaNotFoundError, "Unable to find schema '#{schema}'"
       end
 
+      if Rails.env.test?
+        def find_search!
+          cache.find!("odp/search/v0.1.0/search.json")
+        rescue KeyError
+          raise BopsApi::Errors::SchemaNotFoundError, "Unable to find schema '#{schema}'"
+        end
+      end
```

Comment:
If you put the schema file in `odp/v0.6.0` as `search.json` then you call the following:

``` ruby
BopsApi::Schemas.find!("search", version: "odp/v0.6.0")
```

That way there's no need for the `find_search!` method.